script.js

```diff
@@ -0,0 +1,42 @@
+const inputBox = document.getElementById("name");
+const inputBox2=document.getElementById("email");
+const submitBtn = document.querySelector(".form button");
+const users=document.querySelector(".users");
+const url=' https://reqres.in/api/users';
```

Comment:
avoid unnecessary spacing.

- [ ] remove extra spacing, add single space after = signs wherever required for good redability